Procfile

```diff
@@ -1,4 +1,4 @@
 clock: IS_HEROKU=true npm run start-clock
 release: IS_HEROKU=true npm run on-release
 web: IS_HEROKU=true npm start
-worker: IS_HEROKU=true node --optimize_for_size --max_old_space_size=12000 worker/jobProcessor.js
+worker: IS_HEROKU=true node --optimize_for_size --expose-gc worker/jobProcessor.js
```

Comment:
@iamigo @harshkothari410 I made this change to use the default max_old_space instead of 12GB. We will fail fast if we know this does not work.

Reply:
True